README.md

```diff
@@ -5,3 +5,11 @@ This centralized repository contains all of the automated imports (called _views
 Downstream projects import these views and extend them to suit the needs of the project.
 
 This is the base branch. All generated branches are derived from this branch directly, and files in this branch are kept as-is in the generated branches.
+
+## base branch and why it matters
+
+`base` branch plays an important role in the lookml generation process and should not be deleted!
+
+`base` branch contains some pre-defined `looker-hub` content which should remain and be unchanged after generation. If the generate command overwrites any files found in the `base` branch it exits with an error exit code prior to pushing the changes to this repo.
+
+More detailed description of the lookml generate process can be found [here](https://github.com/mozilla/lookml-generator#generate_command_explained).
```

Comment:
```suggestion
More detailed description of the LookML generate process can be found [here](https://github.com/mozilla/lookml-generator#generate_command_explained).
```
And if you end up tweaking the heading in mozilla/lookml-generator#518 the link here may need to be updated to match.